RemoveDups.c

```diff
@@ -0,0 +1,27 @@
+//Bettina Benitez
+//Sorry Lesley I tried really hard!! but this assignment was too hard :(
```

Comment:
Don't apologize! This is a learning process and I'm glad you tried! The fact that you handed in compiling & running code is awesome enough!